lib/widgets/HomeScreen/home_screen.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:classroom_app/styles/color/colors.dart';
+import 'package:classroom_app/widgets/CustomButton/custom_button.dart';
+
+class HomeScreen extends StatelessWidget {
+  const HomeScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    double screenWidth = MediaQuery.of(context).size.width;
+    double screenHeight = MediaQuery.of(context).size.height;
+    return Scaffold(
+      backgroundColor: AppColors.white,
+      body: Padding(
+        padding: EdgeInsets.symmetric(horizontal: screenWidth * 0.05),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            Center(
+              child: Image.asset(
+                'assets/images/overviewPage_image.png',
+                width: screenWidth * 0.6,
+                height: screenHeight * 0.3,
+                fit: BoxFit.contain,
+              ),
+            ),
+            const SizedBox(
+              height: 30,
+            ),
+            Text(
+              'Make Professional\nYour study plan',
+              textAlign: TextAlign.center,
+              style: TextStyle(
+                  color: AppColors.homeScreenHeadingColor,
+                  fontSize: screenWidth * 0.06,
+                  fontWeight: FontWeight.bold,
+                  fontFamily: 'Poppins'),
+            ),
```

Comment:
The Text widgets have repetitive styles, and the button layout logic is inline.
Extract common styles and widgets into methods or classes for reusability.

```
Widget buildHeadingText(String text, double fontSize, Color color, FontWeight weight) {
  return Text(
    text,
    textAlign: TextAlign.center,
    style: TextStyle(
      color: color,
      fontSize: fontSize,
      fontWeight: weight,
      fontFamily: 'Poppins',
    ),
  );
}
```
```suggestion
            buildHeadingText(
              'Make Professional\nYour study plan',
              screenWidth * 0.06,
              AppColors.homeScreenHeadingColor,
              FontWeight.bold,
            ),
```

Reply:
done